forms/listbox.cpp

```diff
@@ -205,6 +205,24 @@ void ListBox::update()
 	}
 	if (scroller)
 	{
+		size_t mainAxisSize = 0;
+		switch (ListOrientation)
+		{
+			case Orientation::Vertical:
+				mainAxisSize = Size.y;
+				break;
+			case Orientation::Horizontal:
+				mainAxisSize = Size.x;
+				break;
+			default:
+				// shouldn't happen
```

Comment:
If this shows a programming error, maybe a LogWarning (or a LogError if it *can never happen* without magic weird int->enum conversions or memory corruption/un-inited values as there's only 2 values in the enum?)